components/catalogs/front/src/components/ProductMapping/models/ProductMappingSchema.d.ts

```diff
@@ -0,0 +1,15 @@
+export type ProductMappingSchema = {
+    $id: string;
+    $schema: string;
+    $comment: string;
+    title: string;
+    description: string;
+    type: string;
+    properties: {
+        [target: string]: {
+            title?: string;
+            description?: string;
+            type: string;
+        };
+    };
+};
```

Comment:
I think you only need that for now:
```suggestion
export type ProductMappingSchema = {
    properties: {
        [target: string]: {
            title?: string;
        };
    };
};
```